integrations/access/mattermost/bot.go

```diff
@@ -29,7 +29,7 @@ import (
 
 	"github.com/go-resty/resty/v2"
 	"github.com/gravitational/trace"
-	"github.com/mailgun/holster/v3/collections"
+	lru "github.com/hashicorp/golang-lru/v2"
```

Comment:
Some day we should probably drop this dependency too, it's MPL which is acceptable but not preferred.